touchforms/formplayer/static/formplayer/script/deprecate/entrycontrols_type.js

```diff
@@ -355,9 +355,9 @@ function MultiSelectEntry (args) {
     return function () {
       var cbox = $('#ch-' + i);
       if (cbox.is(':checked')) {
-        cbox.removeAttr('checked');
+        cbox.prop('checked', false);
```

Comment:
yeah i also updated the reference to point to deprecate as well. it was more so i could figure out which files were relevant since there were ~9 files that had their code almost duplicated in another file named very similarly. was very confusing. no need to worry about it, i'll take care of deleting them. we'll be removing this submodule soon anyways